sass/_camper_blogs.scss

```diff
@@ -1,16 +1,48 @@
-.camper_blogs {
+.campers_page {
   //background-color: $fccred;
   width: 100%;
   min-height: 200px;
-  padding: 5px 100px 25px 100px;
+  padding: 25px 100px;
   
-  h2 {
-    text-align: center;
+  .subtitle {
+    width: 242px;
+    margin: 0px auto 25px auto;
+    box-sizing: content-box;
+    
+    .toggle {
+      display: inline-block;
+      width: 100px;
+      box-sizing: content-box;
+      
```

Comment:
The `.toggle:hover { cursor: point; }` from below could be nested here like this:

```scss
    &:hover {
        cursor: pointer;
    }
```